functions/supplier/debezium-supplier/pom.xml

```diff
@@ -15,7 +15,7 @@
     <description>Debezium Supplier</description>
 
     <properties>
-        <version.debezium>2.3.0.CR1</version.debezium>
+        <version.debezium>2.3.0.Final</version.debezium>
```

Comment:
I believe we can manage this version in a single place where we would just import `debezium-bom`

Reply:
Yeh, this is a recurring discussion. 
Currently the functions don't have a bom for external dependencies.  The existing `function-dependecies` is only providing a BOM for sharing internal dependencies between the `functions` and `applications`. 
So should we start crating a bom project for evry function common project that might need it or create one shared e.g. something like `function-common-dependecies` bom? 
For either decision is it right to do it at this release stage?

Reply:
Heads up! 
Turned out it is wrong to use `debezium-bom`.
They have too many not project related deps over there.
The point of BOM to manage only your own modules, but not the rest of transitive deps.
🤷